testsuite/cluster-test/src/cluster.rs

```diff
@@ -23,14 +23,15 @@ pub struct Cluster {
 }
 
 impl Cluster {
-    pub fn from_host_port(peers: Vec<(String, u32)>, mint_file: &str) -> Self {
+    pub fn from_host_port(peers: Vec<(String, u32, Option<u16>)>, mint_file: &str) -> Self {
```

Comment:
I think either u32 or u16 are fine for port, but within single crate(cluster test in this case) we want to use same type - it does not make sense that first port in this tuple is u32 and second is u16